base/boot.jl

```diff
@@ -76,7 +76,7 @@
 #end
```

Comment:
the various commented-out types should also be described with the new terminology